frontend/components/customizer/path-builder.tsx

```diff
@@ -229,11 +243,13 @@ export default function PathBuilder({
   }
 
   let namedParamIndex = 0
+  const onlyLiterals =
+    tokens.filter(token => typeof token === 'string').length === tokens.length
```

Comment:
What would you think about using `every` here instead? Believe it'd look something like:
```suggestion
    tokens.every(token => typeof token === 'string')
```

Reply:
Thanks, I was unaware of that method.